source/common/tcp/conn_pool.h

```diff
@@ -178,16 +179,16 @@ class ConnPoolImpl : public Envoy::ConnectionPool::ConnPoolImplBase,
   }
   ConnectionPool::Cancellable* newConnection(Tcp::ConnectionPool::Callbacks& callbacks) override {
     TcpAttachContext context(&callbacks);
-    return newStreamImpl(context);
+    return newStreamImpl(context, /*has_early_data=*/false);
```

Comment:
maybe a comment about why we set this to false. (because we don't yet support TLS early data over TCP)

Reply:
done